styledComponents.js

```diff
@@ -115,7 +115,9 @@ export const ExerciseCard = styled.li`
   position: relative;
   border-radius: 10px;
   overflow: hidden;
-  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  box-shadow:
+    0 4px 8px rgba(0, 0, 0, 0.1),
+    0 -0.5px 5px #0000000d;
```

Comment:
I would try to be consistent with the color types. Here you use RGB once and hex once and further down you write "black" as a word. :wink: